04-cs-fundamentals/classroom/Essential Mathematics.md

```diff
@@ -1,332 +1,208 @@
+_21 \* 3_ = _21 + 21 + 21_ = _63_
+
+Similarly, **exponents can be viewed as repeated multiplication.**</br>
+For example;</br>
+_21 \* 21 \* 21_ = _21<sup>3</sup>_
+
+In the expression, **_21<sup>3</sup>_**, _21_ is what is known as the **base** and _3_ is what is known as the **exponent**. With the base of *21* and exponent of *7*, we have _21<sup>7</sup>_. This can also be read as _twenty one raise to the power of seven_. An exponent tells us to multiply the base to itself that many number of times. e.g.</br></br>
+_21<sup>3</sup>_ = _21 \* 21 \* 21_ = _9261_ </br>
+Read as _twenty one raise to the power of three_.
+
+_21<sup>2</sup>_ = _21 \* 21_ = _441_ </br>
+Read as _twenty one raise to the power of two_.
+
+_21<sup>1</sup>_ = *21* </br>
+Read as _twenty one raise to the power of one_.
 
-### Exercises
-Convert the following decimal numbers to their binary equivalent number using the approach we learned above:
-1. 49
-1. 234
-1. 432
-Check your answers by comparing with the [solutions](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/04-cs-fundamentals/classroom/solutions/decimal_to_binary.md).
+_21<sup>0</sup>_ = *1* </br>
+Read as _twenty one raise to the power of zero_.</br></br>
+**Note**: Any base with an exponent of 0 is equal to 1.
 
-## Further Reading exercise
-Learn how to convert a decimal number to a binary number and vice versa, as well as [Computer Data Basics](https://docs.google.com/document/d/1g-Dxn1rVTChBs9ZZSOgmohjP_RoWNyPJTE-m_n35aZ0/edit). 
+A special form of **exponents** are **square**. For example, _21<sup>2</sup>_ read as _twenty one squared_ means _multiplying a number by itself_.</br>
+When we say, _twenty one cubed_, we mean _21<sup>3</sup>_ i.e. _21 \* 21 \* 21_.</br>
+Learn and revise more about [Exponents on Khan Academy](https://www.khanacademy.org/math/pre-algebra/pre-algebra-exponents-radicals).
 
+**Exponents** and **Logarithms** are closely related. In many ways, we are sharing the same fact in different ways. **Logarithms** are a way of thinking of exponents in the reverse way. To find the exponent, we'd ask a question of the form: _what do we get when we raise twenty one to the power of three?_, which will give us the answer of *9261*. In contrast, with logarithms, we'd ask a question of the form: _how many twenty one does it take to make 9261?_, which will give us the answer of _3_.
 
-<!-- TO DO:  Add the following
-Exponential
+_Whilst exponential expression tells us what we will get if we start with one and keep multiplying the base exponent number of times, logarithmic expressions tell us how many times we will need to keep dividing by the base to get one._</br>
+e.g. If we start with *1* and keep multiplying *2* to itself, *4* times, we are describing _2<sup>4</sup>_, which is *16*. Where as if we start with *16* and keep dividing by *2* until we reach *1*, we are describing _log<sub>2</sub> (16)_, which is *4*.
 
```

Comment:
I'm unable to think of ideas on this one. Did you have anything in mind?

Reply:
Maybe something like this:

The way we write a logarithm is like this:

- Start with the word "log"
- Next is the "base", which is written as a subscript
- Finally, the number that you're taking the logarithm of

In our above example, this would be 3 = log<sub>21</sub>9261

Often times the base is omitted from a logarithm when it "should be" clear from context. In computer science, if you see log without a base, it almost always means log<sub>2</sub>.